src/utils/navigation.ts

```diff
@@ -137,13 +137,18 @@ export const NAVIGATION: Navigation = {
   [config.PRESENTER_WITH_PARAMS_URL]: {
     currentPage: config.PRESENTER_PAGE,
     previousPage: () => config.OVERSEAS_NAME_URL,
-    nextPage: [config.WHO_IS_MAKING_FILING_URL]
+    nextPage: [config.WHO_IS_MAKING_FILING_WITH_PARAMS_URL]
```

Comment:
Not sure this change is needed. A similar one not there on line 130 when we did the presenter/OE name URLs.

Reply:
It does not make a lot of sense to not have this change we would want to navigate to the next page with parameters of we are on one with them. I have tested this locally and we get the parameters in both urls as expected.

Reply:
We didn't have it or need it for the presenter change we did together. I think lets copy that for now (as everyone is copying from the same template) and then we will revisit this later, if required. Please add a subtask to the ROE-1989 story if you are concerned.

Reply:
Just changed this back.